### PR TITLE
Apply Merlin dtypes to Systems code base

### PR DESCRIPTION
**Pull request by karlhigley**
Depends on https://github.com/NVIDIA-Merlin/core/pull/170

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -17,7 +17,8 @@ deps = -rrequirements/test-cpu.txt
 commands =
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/core.git@{posargs:main}
     python -m pip install .
-    
+    python -m pip install --upgrade git+https://github.com/karlhigley/core.git@feature/merlin-dtypes
```
Comment (jperez999) [inline tox.ini:20]: I will approve but I think we want to change this back to use the main branch before integrating right?

**Review by jperez999 — APPROVED** [review]

File: tox.ini [hunk 2]
```diff
@@ -17,7 +17,8 @@ deps = -rrequirements/test-cpu.txt
...
+
```
Comment (karlhigley) [inline tox.ini:21]:
```suggestion
```

File: tox.ini [hunk 3]
```diff
@@ -36,6 +35,7 @@ deps =
 commands =
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/core.git
     python -m pip install .
+    python -m pip install --upgrade git+https://github.com/karlhigley/core.git@feature/merlin-dtypes
```
Comment (karlhigley) [inline tox.ini:38]:
```suggestion
```

File: tox.ini [hunk 4]
```diff
@@ -54,6 +53,7 @@ commands =
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/core.git
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/NVTabular.git
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/models.git
+    python -m pip install --upgrade git+https://github.com/karlhigley/core.git@feature/merlin-dtypes
```
Comment (karlhigley) [inline tox.ini:56]:
```suggestion
```

File: tox.ini [hunk 5]
```diff
@@ -66,7 +65,8 @@ commands =
 ; Install pre-commit-hooks to run these tests during development.
 deps = -rrequirements/dev.txt
 commands =
-    python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/core.git
+    ; python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/core.git
+    python -m pip install --upgrade git+https://github.com/karlhigley/core.git@feature/merlin-dtypes
```
Comment (karlhigley) [inline tox.ini:69]:
```suggestion
    python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/core.git
```

**Pull request merged**